### DISCUSSION TITLE
Imprint/Cookie Banner/ GDPR/DSVGO

### DISCUSSION THREAD

**Discussion by Nazgile94** [pelican-dev/panel#90] [Ideas]
Hi, 
Any possibility the Panel contains the possibility to provide the Google Fonts localy instead over Google directly

- imprint / privacy Page
-  Cookie Banner to opt in / opt out 
-  maybe alternatives to Googles recaptcha 

Providing ptero Panel with publicy Access is a risk Here in EU without modyfing stuff.

Google Fonts, without User Agreement the IP of the Users connection shoudlnt sended to Google (stupid laws Here...)
Instead embedd it localy

Providing a Site without imprint/ privacypage is against the laws , an Editor with HTML Support would BE nice for a simple imprint etc.

Maybe some Sort of Cookie Banner is nice to have too 

Just some ideas to make it more law Proof Outside US :D 

Even If ITS Just for friends... Some random PPL can Access the Site and Report IT to law BCS "my Data IS sended to Google without my Agreement !!! *Im stupid *

Sry for my Bad english, Not my native language.

**Comment by lancepioch** [comment]
Thanks, you will be able to install plugins to accomplish these!
